configure.back

```diff
@@ -0,0 +1,28 @@
+KOKKOS_PATH=$1
```

Comment:
Please remove this file.

Reply:
I removed this file from my repo (https://github.com/yukwangmin/wire-cell-gen-kokkos).

Reply:
Thanks!